lib/Backend/BackwardPass.cpp

```diff
@@ -1818,6 +1908,15 @@ BackwardPass::ProcessBailOutInfo(IR::Instr * instr)
             BVSparse<JitArenaAllocator> * byteCodeUpwardExposedUsed = byteCodeUsesInstr->byteCodeUpwardExposedUsed;
             if (byteCodeUpwardExposedUsed != nullptr)
             {
+                //Stack Args for Formals optimization.
```

Comment:
I guess, you are right. 
Now that I am thinking back, this piece is not required. Let me remove this and add back the tracking for scope object removal - So that i can use this information to decide whether to recreate the scope object or use the existing one in the bail out path.

Currently, I kinda rely on the fact that the scope object will not be restored for sure, because i am removing the bit here - and will always recreate the scope object in the bail out path.

---
In reply to: [68476633](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r68476633) [](ancestors = 68476633,68475602,67800813,67740116)